tuning-primer.sh

```diff
@@ -242,7 +251,11 @@ function second_login_failed()
     read -p "User: " user
     read -rsp "Password: " pass
 
-    local MYSQL_COMMAND_PARAMS="-S $socket -u$user"
+    if [ "$socket" != "" ]; then
+      local MYSQL_COMMAND_PARAMS="-S$socket -u$user"
```

Comment:
Prefer parameter expansion (`${foo[@]}` syntax) where possible.  It's fine if you'd rather not—to put it mildly, there are quite a few spots in the script that break in the face of parameters with spaces and special characters—but I'd prefer new code to follow best practices whenever practical.